src/EditorFeatures/TestUtilities/DocumentationComments/AbstractDocumentationCommentTests.cs

```diff
@@ -156,6 +156,7 @@ private Action CreateInsertTextHandler(ITextView textView, string text)
 
                 execute(
                     view,
+                    Assert.IsType<TestWaitIndicator>(workspace.GetService<IWaitIndicator>()),
```

Comment:
Not sure how I feel about the random testing that implementations are a specific type of mock. This seems like we're just creating debt if/when we restructure helpers.

Reply:
This change is now removed from this pull request.